### ISSUE TITLE
Feature Request: manually binding an agent to a task

### ISSUE THREAD

**Issue by samitherona** [curlyboi/hashtopus#52]
I would like to request a locking mechanism, so that it is possible to override the priority-list for a given agent until the task it it bound to is completed (or stopped / paused somehow).

An example use-case would be: 10 agents, multiple tasks, where 10-n agents work on a very time-consuming task, while the remaining n agents focus on (multiple) "faster" tasks. Once these finish, the agents automatically switch back to the highest priority, in this case the time-consuming task.

**Comment by curlyboi** [comment]
so basically an assignment-specific (task-agent pair) switch that says "stay on this task even if there might be higher priority one"?

**Comment by samitherona** [comment]
Yes, exactly what I was thinking about. Once the tasks finishes, the switch resets and the agent continues normally (requesting the task with the highest priority) again.
